MyCryptoHQ/MyCrypto
# updated tokens

### Description

Updated tokens with [ethereum-lists](https://github.com/ethereum-lists/tokens)

### Changes

* Updated local tokens list using `yarn update:tokens`

### Steps to Test

1. Access account. 
2. Scan for tokens. Verify that token balances show correctly. Verify that there are no odd new entries with excessive balances.
